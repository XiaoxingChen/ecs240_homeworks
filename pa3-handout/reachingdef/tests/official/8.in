c Multiple loops
p 2 8 9 1 8
c bb1: v1 = c;
b 1 1
e 1 2
c bb2: ;
b 2
e 2 3
e 2 5
c bb3: v2 = v1;
b 3 2 1
e 3 4
c bb4: v1 = c;
b 4 1
e 4 2
c bb5: ;
b 5
e 5 6
e 5 8
c bb6: v2 = v1;
b 6 2 1
e 6 7
c bb7: v1 = c;
b 7 1
e 7 5
c bb8: ;
b 8
