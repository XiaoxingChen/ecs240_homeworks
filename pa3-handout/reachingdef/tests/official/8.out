rdout 1 1
rdout 2 1 3 4
rdout 3 1 3 4
rdout 4 3 4
rdout 5 1 3 4 6 7
rdout 6 1 4 6 7
rdout 7 6 7
rdout 8 1 3 4 6 7
